ReactWindows/ReactNative/Views/Picker/ReactPickerManager.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Events;
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace ReactNative.Views.Picker
+{
+    /// <summary>
+    /// A view manager responsible for rendering picker.
+    /// </summary>
+    public class ReactPickerManager : BaseViewManager<ComboBox, ReactPickerShadowNode>
+    {
+        private int _selected;
+
+        /// <summary>
+        /// The name of the view manager.
+        /// </summary>
+        public override string Name
+        {
+            get
+            {
+                return "RCTPicker";
+            }
+        }
+
+        /// <summary>
+        /// Sets whether a picker is enabled.
+        /// </summary>
+        /// <param name="view">a combobox view.</param>
+        /// <param name="enabled">
+        /// Set to <code>true</code> if the picker should be enabled,
+        /// otherwise, set to <code>false</code>.
+        /// </param>
+        [ReactProperty("enabled")]
+        public void SetEnabled(ComboBox view, bool enabled)
+        {
+            view.IsEnabled = enabled;
+        }
+
+        /// <summary>
+        /// Sets the selected item.
+        /// </summary>
+        /// <param name="view">a combobox instance.</param>
+        /// <param name="selected">The selected item.</param>
+        [ReactProperty("selected")]
+        public void SetSelected(ComboBox view, int selected)
+        {
+            // Temporarily disable selection changed event handler.
+            view.SelectionChanged -= OnSelectionChanged;
+
+            _selected = selected;

```

Comment:
Sorry I didn't catch this before... why do you have a private field on the view manager?  The view manager is a singleton instance for the whole app, meaning it manages the state of many ComboBoxes. Not sure this is something you want.